### PR TITLE
Lore-Important Characters Generated Families - Gamerule

### PR DESCRIPTION
**Pull request by MemeWizard**
## Changelog:
- Added Gamerule for #182

### CODE REVIEW

File: localization/english/wc_game_rules_l_english.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿l_english:
+ rule_wc_lore_characters_generate_families:0 "#Bold Warcraft#!: Generate Lore Characters Families"
+ setting_wc_lore_characters_generate_families_on:0 "On"
+ setting_wc_lore_characters_generate_families_on_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will be affected by it."
+ setting_wc_lore_characters_generate_families_off:0 "#high Off#!"
+ setting_wc_lore_characters_generate_families_off_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, Lore Characters will not be affected by it."
```
Comment (zumbak04) [inline localization/english/wc_game_rules_l_english.yml:6]: I would pick only one variation: either Lore Characters or lore characters.

Comment (MemeWizard) [inline localization/english/wc_game_rules_l_english.yml:6]:
```suggestion
 setting_wc_lore_characters_generate_families_off_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will not be affected by it."
```

File: localization/english/wc_game_rules_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿l_english:
+ rule_wc_lore_characters_generate_families:0 "#Bold Warcraft#!: Generate Lore Characters Families"
+ setting_wc_lore_characters_generate_families_on:0 "On"
+ setting_wc_lore_characters_generate_families_on_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will be affected by it."
+ setting_wc_lore_characters_generate_families_off:0 "#high Off#!"
```
Comment (zumbak04) [inline localization/english/wc_game_rules_l_english.yml:5]: I thought that the default option shouldn't be high.
<details>
<summary>Click to expand</summary>

image
image

</details>

Comment (MemeWizard) [inline localization/english/wc_game_rules_l_english.yml:5]:
```suggestion
 setting_wc_lore_characters_generate_families_off:0 "Off"
```

File: localization/english/wc_game_rules_l_english.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿l_english:
+ rule_wc_lore_characters_generate_families:0 "#Bold Warcraft#!: Generate Lore Characters Families"
+ setting_wc_lore_characters_generate_families_on:0 "On"
```
Comment (MemeWizard) [inline localization/english/wc_game_rules_l_english.yml:3]:
```suggestion
 setting_wc_lore_characters_generate_families_on:0 "#high On#!"
```

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by ercarp — APPROVED** [review]

**Pull request merged**